pennylane_qiskit/qiskit_device.py

```diff
@@ -82,29 +82,26 @@ def pauli_eigs(n):
     "RZ": ex.RZGate,
     "S": ex.SGate,
     "T": ex.TGate,
+
+    # Adding the following for conversion compatibility
+    "Sdg": ex.SdgGate,
+    "Tdg": ex.TdgGate,
```

Comment:
How come these are kept?

Reply:
Great one, thanks for that! Updated it in the latest commit